packages/stark-core/src/logging/entities/log-message.entity.intf.ts

```diff
@@ -1,11 +1,12 @@
 "use strict";
 
 import { StarkLogMessageType } from "./log-message-type.entity";
+import { StarkError } from '../../common';
```

Comment:
Are we sure this exports are working fine with Rollup? Could you double check?
As far as I remember we have to add the "/index" in the path so that Rollup can take the index.ts file correctly